objectRecognitionSource/src/test/java/io/vantiq/extsrc/objectRecognition/neuralNet/TestYoloProcessor.java

```diff
@@ -120,6 +120,63 @@ public void testResults() throws ImageProcessingException {
             fail("Could not interpret json string" + e.getMessage());
         }
     }
+    
+    @Test
+    public void testValidAnchors() throws ImageProcessingException {
+        Map config = new LinkedHashMap<>();
+        YoloProcessor ypImageSaver = new YoloProcessor();
+        
+        List<Number> anchorList = Arrays.asList(1,1,1,1,1,1,1,1,1,1);
```

Comment:
We should have a test that verifies that floating point works, too.  Based on how the code works, a single array of mixed numbers should work.

Also -- seems like we need a test that verifies that a "real" config document -- with a JSON array of either numbers or character strings (if that's accepted) will work.